certz/README.md

```diff
@@ -54,6 +54,41 @@ When no longer a profile is needed it can be removed from the target via
 `Certz.DeleteProfile()` RPC. Note that the gNxI SSL profile cannot be
 removed.
 
+#### Authentication Policy
+
+An authentication policy is a set of rules that defines which CAs can sign which
+certificate. By rotating authentication policies, data center admins can ensure
+that only authorized CAs are able to sign certificates for specific users/roles
+in the authentication framework. This helps to minimize the impact of a security
+breach, as it prevents, for example, an attacker from using a less privileged CA
+to sign for high value users/roles.
+
+##### Details
+
+When a client tries to establish a gRPC connection to a gRPC server, the server
+must verify that the client is authorized to do so. To do that the client
+presents a certificate to the server, which the server verifies to see if it was
+issued by a trusted Certificate Authority (CA).
+
+In an enterprise environment, it is impractical and risky to have all client
+certificates signed by a single CA. It is also impractical to provide the gRPC
+server with certificates from all CAs that can issue client certificates.
+
+The solution to this problem is to have a single "root of trust" that is common
+to all CAs. This root of trust can be used to verify certificates issued by all
+intermediate CAs. In such case, the gRPC server only needs to be configured with
+the “root of trust” certificate. The client must provide a chain of certificates
+that includes certificates from all intermediate CAs.
+
+This organization of certificate issuing processes also allows for
+specialization of CAs. It is possible to limit which CAs can issue certificates
+for particular users. This allows the gRPC server to reject certificates that
+are otherwise valid but were signed by the wrong CA.
+
```

Comment:
It feels to me that the two paragraphs above are a bit too prescriptive in the context of these APIs.  Suggest to remove.

Reply:
LGTM. Changed to your version of this paragraph.